src/main/scala/LSWidgets.scala

```diff
@@ -126,6 +95,13 @@ class ProcedureWidget(val key: WidgetKey, val state: State, val ws: GUIWorkspace
       override def removeUpdate(e: DocumentEvent): Unit = updateInState(property)
       override def insertUpdate(e: DocumentEvent): Unit = updateInState(property)
     })
+
+  private def boundTextField(i: Int, prop: StringProperty[this.type]) = {
+    val t = new JTextField(i)
```

Comment:
Why set the number of columns at all? I did that for the name field to limit its width, but that's not necessary anymore.